piksi_tools/console/settings.yaml

```diff
@@ -2564,7 +2566,7 @@
   expert: true
   type: string
   units: N/A
-  default value: '72,74,117,522,527'
+  default value: '23,65,72,74,81,97,117,134,136,137,138,139,144,149,163,165,166,167,171,175,181,185,187,188,189,190,257,258,259,520,522,524,526,527,528,1025,2304,2305,2306,30583,65280,65282,65535'
```

Comment:
If these are the CAN ports (reviewing from mobile), I don’t think this is correct.  The CAN ports had an intentionally much more restrictive set of defaults.

Reply:
I took these from https://github.com/swift-nav/piksi_buildroot/blob/master/package/ports_daemon/ports_daemon/src/whitelists.c, I can just remove from this PR if you prefer.

Reply:
Ok, either way works, that's much more enabled messages than I expected it to have.

Reply:
I'll remove for now.